packages/react-params/src/initValue.ts

```diff
@@ -140,7 +138,9 @@ export default function getInitValue (registry: Registry, def: TypeDef): unknown
       if (!warnList.includes(type)) {
         warnList.push(type);
         error && console.error(`params: initValue: ${error}`);
-        console.info(`params: initValue: No default value for type ${type} from ${JSON.stringify(def)}, using defaults`);
+        console.info(
+          `params: initValue: No default value for type ${type} from ${JSON.stringify(def)}, using defaults`
+        );
```

Comment:
```suggestion
        console.info(`params: initValue: No default value for type ${type} from ${JSON.stringify(def)}, using defaults`);
```

Prettier? Very weird one.